docs/source/TetraOpt_Algorithm/Parameters_and_Hyperparameters.md

```diff
@@ -1,23 +1,26 @@
+| `maximal_rank`                   | int     | Maximal rank for TetraOpt.                                                                                                                                                                                          |
+| `iteration_number`               | int     | Maximal iteration number for TetraOpt.                                                                                                                                                                              |
+| `quantization`                   | boolean | Whether or not to use quantization, or virtual dimension as it is called elsewhere. Using virtual dimension allows tensors to be reshaped if their sizes are of composite integers.                                 |
+| `points_number`                  | int     | Determines the number points to discard after every iteration.                                                                                                                                                      |
+| `tolerance`                      | float   | Desired tolerance for the algorithm.                                                                                                                                                                                |
+| `upper_limits`                   | array   | Set upper limits for the objective function arguments.                                                                                                                                                              |
+| `lower_limits`                   | array   | Set lower limits for the objective function arguments.                                                                                                                                                              |
+| `grid`                           | array   | Number of grid points for the objective function arguments to optimize. TetraOpt performs an optimization over a discretized space, whose grid is determined by this parameter.                                     |
+| `objective_function`             | URL     | This is the URL for the endpoint defining the objective function.                                                                                                                                                   |                                                                                                                                                  | 
+| `local_optimizer`                | URL     | This is the URL for the endpoint defining the local optimizer function.                                                                                                                                             | 
```

Comment:
Please indicate the following:
- Only can choose one of objective function or objective function channel id (1)
- only choose up to one of local optimizer or local optimizer channel id (0 or 1)